src/proposals/20250220/SparkEthereum_20250220.t.sol

```diff
@@ -239,61 +243,128 @@ contract SparkEthereum_20250220Test is SparkTestBase {
 
         assertEq(controller.mintRecipients(CCTPForwarder.DOMAIN_ID_CIRCLE_ETHEREUM), bytes32(uint256(uint160(Ethereum.ALM_PROXY))));
     }
+
     function test_ETHEREUM_ARBITRUM_sparkLiquidityLayerE2E() public onChain(ChainIdUtils.Ethereum()) {
         executeAllPayloadsAndBridges();
 
+        IERC20 arbSUsds = IERC20(Arbitrum.SUSDS);
+        IERC20 arbUsdc  = IERC20(Arbitrum.USDC);
+        IERC20 arbUsds  = IERC20(Arbitrum.USDS);
+
+        MainnetController mainnetController = MainnetController(Ethereum.ALM_CONTROLLER);
+        ForeignController arbController     = ForeignController(Arbitrum.ALM_CONTROLLER);
+
+        // NOTE: These values are calculated on Ethereum and used in Arbitrum, inherently
+        //       asserting that the sUSDS exchange rates match.
```

Comment:
It actually doesn't assert this. This is just to get a number that is around 10m in value in shares. The SSR could be totally off and this still works. You would just get different shares in the PSM.

In fact we should test this to make sure the SSR oracle is good.